test/functional/operation_example.test.js

```diff
@@ -9354,9 +9354,7 @@ describe('Operation Examples', function() {
    * @ignore
    */
   it('Should correctly add capped collection options to cursor', {
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
-    },
+    metadata: { requires: { os: '!win32' } }, // leaks on windows
```

Comment:
Can you make a ticket to track all of these tests which "leak on windows", and leave a reference to them in the code where the tests are skipped? This means that streams, or cursors (or both) are kind of broken on windows. Someone should look into that relatively soon